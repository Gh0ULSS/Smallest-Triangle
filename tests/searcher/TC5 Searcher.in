24.73, 36.5
-21.41, 46.56
25.49, 42.92
-22.59, -13.34
23.56, -13.65
-47.48, 17.51
-44.84, -26.53
26.14, -26.58
24.04, -13.65
-33.59, -42.64
45.52, -48.87
23.17, -13.65