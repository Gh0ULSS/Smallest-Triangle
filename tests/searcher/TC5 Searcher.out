read 12 points
23.56, -13.65
23.17, -13.65
24.04, -13.65
This is not a triangle